tests/units/anta_tests/test_security.py

```diff
@@ -569,4 +571,78 @@
             ],
         },
     },
+    {
+        "name": "success",
+        "test": VerifyBannerLogin,
+        "eos_data": [
+            {
+                "loginBanner": "Copyright (c) 2023-2024 Arista Networks, Inc.\nUse of this source code is governed by the Apache License 2.0\n"
+                "that can be found in the LICENSE file."
+            }
+        ],
+        "inputs": {
+            "login_banner": "Copyright (c) 2023-2024 Arista Networks, Inc.\nUse of this source code is governed by the Apache License 2.0\n"
+            "that can be found in the LICENSE file."
+        },
+        "expected": {"result": "success"},
+    },
+    {
+        "name": "failure-incorrect-login-banner",
+        "test": VerifyBannerLogin,
+        "eos_data": [
+            {
+                "loginBanner": "Copyright (c) 2023 Arista Networks, Inc.\nUse of this source code is governed by the Apache License 2.0\n"
```

Comment:
can you test with a multiline banner entry?
```
"""
opyright (c) 2023-2024 Arista Networks, Inc.
Use of this source code is governed by the Apache License 2.0
"""
```

Reply:
In device json output will be same as eos_data so i add the support if i user want to give multiline banner as input.
32ef660